src/chainstate/coordinator/tests.rs

```diff
@@ -293,8 +295,13 @@ pub fn get_chainstate_path(path: &str) -> String {
 }
 
 pub fn get_chainstate(path: &str) -> StacksChainState {
-    let (chainstate, _) =
-        StacksChainState::open(false, 0x80000000, &get_chainstate_path(path)).unwrap();
+    let (chainstate, _) = StacksChainState::open(
+        false,
+        0x80000000,
+        &get_chainstate_path(path),
+        MAXIMUM_MEMPOOL_TX_CHAINING,
```

Comment:
If we are dealing with a constant, why do we need to pass it in constructors? I'd be in favor of keeping this constructor as lean as possible.

Reply:
Oh I see we can override using the settings. Isn't something that @jcnelson pushed back on?

Reply:
Yes, I don't think this should be a setting.  I think that correct nodes should always permit a _minimum_ degree of transaction-chaining that is equal to the constant `MAXIMUM_MEMPOOL_TX_CHAINING`.  If node operators have a particular reason to permit longer chains, they can always re-compile with a bigger value.

Reply:
will remove